helm/solr/templates/solrcloud.yaml

```diff
@@ -108,6 +108,12 @@ spec:
     {{- end }}
   {{- end }}
 
+  {{- if and (.Values.availability) (.Values.availability.podDisruptionBudget) }}
```

Comment:
I don't think you need this `and`... I think`.Values.availability.podDisruptionBudget` will return false if `.Values.availability` doesn't exist.

Reply:
I actually tried that first - I get a helm parse error.

```
$ helm template .
Error: template: solr/templates/solrcloud.yaml:111:16: executing "solr/templates/solrcloud.yaml" at <.Values.availability.podDisruptionBudget>: nil pointer evaluating interface {}.podDisruptionBudget
```

Reply:
Ahh nice, that makes sense!